src/rfac/rfrp.c

```
@@ -0,0 +1,89 @@
+ *
+ * \param[in] eng pointer to list of energies.
+ * \param[in] e_int pointer to list of experimental intensities.
+ * \param[in] t_int pointer to list of theoretical intensities.
+ *
+ * \warning \p eng must be terminated by F_END_OF_LIST.
+ *
+ * \param vi Imaginary part of the optical potential.
+ *
+ * \return Rp if successful.
+ */
+real rfac_rp(const real *eng, const real *e_int, const real *t_int, real vi)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'rfac_rp' of similar type ('const int *') are easily swapped by mistake

(bugprone-easily-swappable-parameters)